static/src/javascripts/es6/projects/common/modules/crosswords/helpers/get-intersecting-entries.js

```diff
@@ -8,7 +8,7 @@ export default (entries, focussed) => {
     const entryHasIntersectingCell = entry => {
         const cells = helpers.cellsForEntry(entry);
         const intersecting = deepIntersection(cells, focussedCells);
-        return !! intersecting.length;
+        return !!intersecting.length;
```

Comment:
:unamused: I actually prefer the former because my eyes sometimes skip over the operand. Is there a rule we can set?